installer/install.iss

```diff
@@ -1133,7 +1133,11 @@ begin
 
     EditorAvailable[GE_NotepadPlusPlus]:=RegQueryStringValue(HKEY_LOCAL_MACHINE,'SOFTWARE\Microsoft\Windows\CurrentVersion\App Paths\notepad++.exe','',NotepadPlusPlusPath);
     EditorAvailable[GE_VisualStudioCode]:=RegQueryStringValue(HKEY_LOCAL_MACHINE,'SOFTWARE\Classes\Applications\Code.exe\shell\open\command','',VisualStudioCodePath);
+    if (not EditorAvailable[GE_VisualStudioCode]) then
+        EditorAvailable[GE_VisualStudioCode]:=RegQueryStringValue(HKEY_CURRENT_USER,'Software\Classes\Applications\Code.exe\shell\open\command','',VisualStudioCodePath);
```

Comment:
Question 1: does this even work? Git for Windows' installer runs elevated, so I could imagine that this change makes the installer look in the *administrator's* `HKEY_CURRENT_USER` namespace. And that would not catch the original user's VS Code installation, right?

Question 2 (or actually, more of a very strong suggestion): isn't it inappropriate for a system-wide installation of Git for Windows to depend on a user-specific VS Code installation? In general, other users would not even be able to access that VS Code, and even then, it would wreak havoc if the original user wants to update their VS Code and another user still uses it. So if we go the route at all to detect a user setup of VS Code, I am fairly convinced that we should *not* install that via `git config --system core.editor <path>` but instead via `git config --global core.editor <path>` (of course, launched via [`ExecAsOriginalUser()`](http://www.innosetup.com/ishelp/topic_isxfunc_execasoriginaluser.htm)). This will complicate the code quite a bit, though, as we now have to record not only the path but also the fact that VS Code is to be configured for the current user only.

Reply:
Answer 1: Yes, the installer needs to look into the user's HKCU.

Answer 2: It does seem a bit backwards. But if done, it should definitely go into the user's `--global`.

Reply:
The problem with answer 2: we now have no longer any way to determine whether the setting in `~/.gitconfig` was configured by the installer (and should hence be removed upon uninstall), or by the user. Do you have any ideas how to solve that conundrum?